### PR TITLE
PIP-1598-pbam-bulk-rna

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: accession/accession.py [hunk 1]
```diff
@@ -1246,6 +1246,60 @@ def make_mad_qc_metric(self, encode_file: EncodeFile, file: File) -> None:
...
+        self,
+        encode_file: EncodeFile,
+        file: File,
+        task_output_name: str,
+        qc_dictionary_key: str,
+        convert_to_string: List[str,] = [
+            "mapped_pct",
+            "paired_properly_pct",
+            "singletons_pct",
+        ],
```
Comment (paul-sud) [inline accession/accession.py]: It is a mutable default argument. Does this really need to be exposed as an argument? Could maybe just put it in the body of the function.

Comment (paul-sud) [reply accession/accession.py]: Otherwise could just make it a tuple.

**Review by paul-sud — CHANGES_REQUESTED** [review]
One change requested. Also will need to use the new step versions with `ptools-bin` included.

One general comment - if you think it's useful, there is a feature that avoids posting the GCS URIs for the files and also doesn't post the metadata JSON as analysis attachment. We do it for ChIP pbam, it's set by passing `private_filenames=True` to `Accession` constructor. It does require complicating the accession factory though.

File: accession/accession.py (hunk 1 above)
Comment (ottojolanki) [reply accession/accession.py]: yeah, that makes sense.

**Review by paul-sud — APPROVED** [review]
LGTM

**Pull request merged**